documentation/master-devonfw-dashboard.asciidoc

```diff
@@ -2,24 +2,24 @@
 
 == Landing page
 
-include::landing-page[leveloffset=2]
+include::landing-page.asciidoc[leveloffset=2]
 
 == Home
 
-include::home-page[leveloffset=2]
+include::home-page.asciidoc[leveloffset=2]
 
 == Projects
 
-include::projects-page[leveloffset=2]
+include::projects-page.asciidoc[leveloffset=2]
 
 == Repositories
 
-include::repositories-page[leveloffset=2]
+include::repositories-page.asciidoc[leveloffset=2]
 
 == Wiki
 
-include::wiki-page[leveloffset=2]
+include::wiki-page.asciidoc[leveloffset=2]
 
 == Settings
 
-include::settings-page[leveloffset=2]
+include::settings-page.asciidoc[leveloffset=2]
```

Comment:
In include statements you don't have to add .asciidoc

https://raw.githubusercontent.com/devonfw/devon4ng/develop/documentation/master-devon4ng.asciidoc

Reply:
If I remove `.asciidoc`, it shows an error in the preview: Unresolved directive in <stdin> - include::landing-page[leveloffset=2]

Will it work as expected in the Wiki without `.asciidoc`?

Reply:
> If I remove `.asciidoc`, it shows an error in the preview: Unresolved directive in - include::landing-page[leveloffset=2]
> 
> Will it work as expected in the Wiki without `.asciidoc`?

You can check and example of master file in devon4ng wiki: https://github.com/devonfw/devon4ng/wiki/master-devon4ng. In there, it is being shown properly.

Even if it were not being displayed correctly, master-*.asciidoc are only used by the website generator (for the wiki we already have sidebar.asciidoc) so I prefer it to keep master-devonfw-dashboard.asciidoc like the other master-*.asciidoc files

Reply:
Corrected.